web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -110,6 +111,14 @@ export enum DiscoverEventStatus {
   Aborted = 'DISCOVER_STATUS_ABORTED', // user exits the wizard
 }
 
+export enum CtaEvents {
```

Comment:
Isn't it possible to use the enum from `gen/proto/js/prehog/v1alpha/teleport_pb.js` here?

Reply:
It is, and its much better, thank you!
Added in https://github.com/gravitational/teleport/pull/25325/commits/b07bdf88feccdc3d9dda03e04a47a2df827f1967.

I've exported it from the types file, I think it makes sense. Let me know what you think.